cli/cliui/table_test.go

```diff
@@ -12,6 +13,16 @@ import (
 	"github.com/coder/coder/cli/cliui"
 )
 
+type stringWrapper struct {
+	str string
+}
+
+var _ fmt.Stringer = stringWrapper{}
+
+func (s stringWrapper) String() string {
+	return s.str
+}
+
```

Comment:
`string` doesn't implement `fmt.Stringer`? 🥴

Reply:
strings will be rendered just fine by themselves, but I wanted to test that stringers get expanded properly